python/mxnet/ndarray/numpy/_op.py

```diff
@@ -39,7 +39,7 @@
            'around', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal', 'less_equal',
            'hsplit', 'rot90', 'einsum', 'true_divide', 'nonzero', 'shares_memory', 'may_share_memory', 'diff', 'resize',
-           'nan_to_num']
+           'nan_to_num', 'full_like']
```

Comment:
move this to the position after `full` in the list. Same applies to all other files.

Reply:
Also remember `zeros_like` and `ones_like`, put them after `zeros_like` and `ones_like` respectively.